### DISCUSSION TITLE
Get the response even if the status code doesn't match the expected

### DISCUSSION THREAD

**Discussion by diegozklein** [pactumjs/pactum#304] [Q&A]
There's any way to get the error response message in a more "readable" way?

I know that the response is printed in the console every time, but I want to know if there's a way to log the response in a "better" way.

**Comment by cbruchwaldnetfonds** [comment]
If you use expect() the third prop is a custom message you can handle in.

**Reply by diegozklein** [reply]
I see, I will try that way, thank you 👍
